addons/binding/org.openhab.binding.fritzaha/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="avmfritz"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 org.eclipse.smarthome.thing-description.xsd">
+
+	<bridge-type id="fritzbox">
+		<label>FRITZ!Box AHA</label>
+		<description>This bridge represents a Fritz!-Box as gateway to connected AHA devices.
+		</description>
+
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<context>network-address</context>
+				<label>IP address</label>
+				<description>The local IP address or hostname of the FRITZ!Box
+				</description>
+				<default>fritz.box</default>
```

Comment:
Especially for the manual setup I would prefer the user to provide an IP address - I have seen to many situations where fritz.box does not work as expected, e.g. because you have DNS resolution problems on your gateway or you have two fritzboxes in your network etc...